File: Modulo_I/Examen Practico/Correccion Examen/usuario.php
<?php

class Usuario{

	private $nombre;
	private $password;
	private $estado;
	private $rutaImagen;
	
	function __construct($nombre,$password,$estado,$rutaImagen){

		$this->nombre=$nombre;
		$this->password=$password;
		$this->estado=$estado;
		$this->rutaImagen=$rutaImagen;
	}
	

	function getNombre(){
		return $this->nombre;
	}
	function getPassword(){
		return $this->password;
	}
	function getEstado(){
		return $this->estado;
	}
	function getRutaImagen(){
		return $this->rutaImagen;
	}
	function setEstado($estado){
		$this->estado=$estado;
	}
}

?>

File: Modulo_II/JEE/GestorTareas/src/beans/Usuario.java
package beans;

import java.io.Serializable;

import javax.persistence.Entity;
import javax.persistence.Id;


@Entity
public class Usuario implements Serializable{
	
	@Id
	private String nombre;
	private String password;
	
	public Usuario(){}

	public Usuario(String nombre, String password) {
		super();
		this.nombre = nombre;
		this.password = password;
	}

	public String getNombre() {
		return nombre;
	}

	public void setNombre(String nombre) {
		this.nombre = nombre;
	}

	public String getPassword() {
		return password;
	}

	public void setPassword(String password) {
		this.password = password;
	}
	

}
